Tests/Unit/LoRaWanTests/LinkedTimeoutCancellationTokenTests.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Unit.LoRaWanTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using Xunit;
+
+    public class LinkedTimeoutCancellationTokenTests
+    {
+        /*[Fact]
+        public void Equality()
+        {
```

Comment:
What about this commented code?

Reply:
Yes, just clarified with @bastbu, who said he couldn't get it to work. I'll address it.

Reply:
It was less about getting it to work and more about what kind of equality we want to implement on the `LinkedTimeoutCancellationToken`. Thanks for addressing it.